CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.9.0]
+### Added
+- Add a new `attemptMakeApp` method which allows clients to attempt a creation of the Ledger transport for the purposes of detecting/catching potential connection errors.  ([#126](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/126))
+
+### Changed
+- Update `hw-eth-app` dependency to `^6.12.2`. This fixes a bug in the Ledger eth-app signing code that could cause failures when trying to sign EIP-1559 transactions on networks with high chain ids.  ([#127](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/127))
+
```

Comment:
This isn't a change to the keyring - it's a change to the bridge, and not tied to this version. I don't think we should include bridge changes as part of this changelog. It would give a false impression of what this release consists of.

```suggestion

```

Reply:
Thanks. Bad mistake on my part.